static/src/javascripts/projects/commercial/modules/article-body-adverts.spec.js

```diff
@@ -40,7 +40,7 @@ jest.mock('lib/detect', () => ({
 }));
 jest.mock('lib/config', () => ({ page: {} }));
 
-const spaceFillerStub: JestMockFn = (spaceFiller.fillSpace: any);
+const spaceFillerStub: JestMockFn<*, *> = (spaceFiller.fillSpace: any);
```

Comment:
Apparently the generics here allow us to verify that the arguments passed to the function and the return value match the expected types. Perhaps we should utilise this feature instead of simply inferring types using the `*` syntax